Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/NetworkSearchController.java

```diff
@@ -150,4 +150,48 @@ public String getSearchTransformedNetwork(@PathVariable("projectUnixName") Strin
         return "sites/networks/searchednetwork";
     }
 
+
+    @RequestMapping(value = "sites/networks/search", method = RequestMethod.GET)
+    public String getSearchTransformedNetwork(@RequestParam("conceptId") String conceptId,@RequestParam("projectName") List<String> projectNames,
```

Comment:
and same here, variable names should reflect what they hold and their type. I'm sure you don't want a list of project names. Probably just a list of project ids.